docs/tutorial-react-native/start-streaming.mdx

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 3
+---
+
+# Start streaming
+
+How to stream your camera
+
+:::tip Enable devices before connecting
```

Comment:
**suggestion:** 
```suggestion
:::tip Enable camera and microphone before connecting
```

It's not really a tip but rather a requirement in this case 🤔